common/src/main/proto/TransportMessages.proto

```diff
@@ -278,6 +282,26 @@ message PbGetReducerFileGroupResponse {
   repeated int32 partitionIds = 4;
 }
 
+message PbGetShuffleId {
```

Comment:
Does the `TransportMessage#getParsedPayload` need to add these message?

Reply:
can you help to explain what kind of message needs to be added to TransportMessage#getParsedPayload?

Reply:
@ErikFang, all PB messages should use `TransportMessage#getParsedPayload` to parse the payload.